### ISSUE TITLE
Considering Nimble commander

### ISSUE THREAD

**Issue by EricEWeir** [mikekazakov/nimble-commander#131]
After many years with Path Finder I'm done and looking for a replacement. I came close to buying Forklift but hesitated. Took and initial look at Nimble Commander. Liked that it's clean and quick and command driven, but hesitated again. Now seriously considering it but have a reservation. 

I am not a developer or a coder of any kind. I've been a writer but now am retired. Wonder about receptivity to a non-techie like me. I might have questions or requests with which established Nimble Commander users might be impatient. 

I'm capable of learning. Especially with help. Which may turn out not to be all the necessary after all. 

So, will I be welcome in this crowd?

Thanks,

Eric

**Comment by mikekazakov** [comment]
Hi Eric,
Thanks for the interest in NC!

This year my goal is to transform NC to be a fully open source and free of charge project.
As a part of this process of course I'd like to be as open and welcoming to community as possible.
How will it go - only time will tell I guess :)
So please feel free to ask, I'll try to help.
There's now a Q&A section in the Discussions - what would be the best place.

Thanks,
Michael

**Comment by EricEWeir** [comment]
> On Jan 31, 2024, at 3:29 PM, Michael G. Kazakov ***@***.***> wrote:
> 
> ﻿Hi Eric,
> Thanks for the interest in NC!
> 
> This year my goal is to transform NC to be a fully open source and free of charge project.
> As a part of this process of course I'd like to be as open and welcoming to community as possible.
> How will it go - only time will tell I guess :)
> So please feel free to ask, I'll try to help.
> There's now a Q&A section in the Discussions - what would be the best place.
> 

Thanks, Michael.

If there is currently a subscription fee I’ll happily pay it. 


Regards,
———————————
Eric Weir 
Atlanta, GA  USA

“It has all been combustion.”

- W.G. Sebald

**Comment by mikekazakov** [comment]
Yes, it's currently available on both Mac App Store and Fast Spring - go https://magnumbytes.com/.
I'll move this thread into discussions.
